pkg/crd/markers/zz_generated.markerhelp.go

```diff
@@ -24,6 +24,22 @@ import (
 	"sigs.k8s.io/controller-tools/pkg/markers"
 )
 
+func (Default) Help() *markers.DefinitionHelp {
+	return &markers.DefinitionHelp{
+		Category: "CRD validation",
+		DetailedHelp: markers.DetailedHelp{
+			Summary: "sets the default value for this field. ",
+			Details: " A default value will be accepted as any valid default value (e.g. boolean: `true`, string: `Cluster`, numerical: `1.24`, array: `{1,2}`, object: `{policy: \"delete\"}`) and validated as the type indicated for the field.",
```

Comment:
leading space?

Reply:
Maybe document that this must be pruned and validate against the schema.

Reply:
The leading space seems to be common to all generated help. @DirectXMan12 is this a bug?

Reply:
> Maybe document that this must be pruned and validate against the schema.

Done